Test/TestsDatabase/OrderDatabaseTests.cs

```diff
@@ -0,0 +1,173 @@
+﻿using System;
+using System.Linq;
+using Anlab.Core.Domain;
+using AnlabMvc.Data;
+using Microsoft.Data.Sqlite;
+using Microsoft.EntityFrameworkCore;
+using Microsoft.EntityFrameworkCore.Internal;
+using Shouldly;
+using Test.Helpers;
+using Xunit;
+
+namespace Test.TestsDatabase
+{
+    [Trait("Category", "DatabaseTests")]
+    public class OrderDatabaseTests
+    {
+        [Fact]
+        public void test1()

```

Comment:
@srkirkland Yes, but:
Note 2: The xUnit.net team feels that per-test setup and teardown creates difficult-to-follow and debug testing code, often causing unnecessary code to run before every single test is run. For more information, see http://jamesnewkirk.typepad.com/posts/2007/09/why-you-should-.html.